traffic_ops/app/lib/UI/Utils.pm

```diff
@@ -413,4 +414,81 @@ sub current_user_tenant {
     return $self->db->resultset('TmUser')->search( { username => $self->current_user()->{username} } )->get_column('tenant_id')->single();
 }    
 
+sub verify_tenancy {
```

Comment:
I would suggest that this method is never called inside a loop. for example, when looping thru a query result set. the 2 queries performed in this method will cause problems when TO is setup with a remote postgres database.